### PR TITLE
fix(wc1): Fix tokens, variables, styles

### PR DESCRIPTION
**Pull request by mariannuar**
## Summary

This PR fixes the following 

- Way of usen tokens/variables in some components
- Styles

## How to review this pull request
- [ ] Run `npm run develop`
- [ ] Check all the current stories look good
- [ ] Check all components are using the `tokens/variables` instead of the `data.value`

### CODE REVIEW

File: src/components/tokens/color-tokens/color-tokens.ts [hunk 1]
```diff
@@ -12,48 +12,44 @@ export class ColorTokens extends LitElement {
...
-                  <span>
-                    ${token}
-                    <span>
-                      <code class="sb-custom-property-name"
-                        >var(--color-${token})</code
-                      >
-                    </span>
+      ([token, data]) => {
+        const newToken = token.replace(/\B([A-Z])\B/g, '-$1').toLowerCase();
+        console.log(Object.entries(data));
```
Comment (mcortes19) [inline src/components/tokens/color-tokens/color-tokens.ts]: @mariannuar please get rid of this `console.log`

**Review by mcortes19 — CHANGES_REQUESTED** [review]
@mariannuar left a comment related to a console.log I found, other than that everything looks good to me!

**Comment by mariannuar** [comment]
@mcortes19 Done. Thank you!

**Review by mcortes19 — APPROVED** [review]
@mariannuar LGTM 🎖️

**Pull request merged**